internal/handlers/pg/msg_getparameter.go

```diff
@@ -25,25 +26,80 @@ import (
 
 // MsgGetParameter OpMsg used to get parameter.
 func (h *Handler) MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	document, err := msg.Document()
+	cmd, err := msg.Document()
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/449
+	getParameter, err := cmd.Get("getParameter")
```

Comment:
That's typically written as `command := document.Command()`

Reply:
`command := document.Command()` this is getting the command name
and in this case I get the value of the command

Reply:
That's true, but then we should use `document.Get(command)`, not `command.Get("getParameter")`, for consistencty